core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildLocalContainerRunner.java

```diff
@@ -44,16 +40,19 @@ public NativeImageBuildLocalContainerRunner(NativeConfig nativeConfig, Path outp
 
     @Override
     protected List<String> getContainerRuntimeBuildArgs() {
-        List<String> containerRuntimeArgs = super.getContainerRuntimeBuildArgs();
-        String volumeOutputPath = outputPath;
+        final List<String> containerRuntimeArgs = super.getContainerRuntimeBuildArgs();
+        final String volumeOutputPath;
         if (SystemUtils.IS_OS_WINDOWS) {
-            volumeOutputPath = FileUtil.translateToVolumePath(volumeOutputPath);
+            volumeOutputPath = FileUtil.translateToVolumePath(outputPath);
+        } else {
+            volumeOutputPath = outputPath;
         }
 
-        String selinuxBindOption = ":z";
-        if (SystemUtils.IS_OS_MAC
-                && ContainerRuntimeUtil.detectContainerRuntime() == ContainerRuntimeUtil.ContainerRuntime.PODMAN) {
+        final String selinuxBindOption;
+        if (SystemUtils.IS_OS_MAC && containerRuntime == PODMAN) {
             selinuxBindOption = "";
```

Comment:
do we know the reason why podman on mac needs different arguments here?

Reply:
No idea. I don't have access to any mac hw, so I tested this PR only on Windows and Linux. This bit hasn't been changed though.

Reply:
mac qemu doesnt support selinux labels over 9pfs.  Although not sure if this leads to an actual error, if it does it could be changed to ignore